crates/nu-command/src/conversions/into/filesize.rs

```diff
@@ -107,6 +107,11 @@ impl Command for SubCommand {
                 example: "4KB | into filesize",
                 result: Some(Value::test_filesize(4000)),
             },
+            Example {
+                description: "Convert negative file size to filesize",
+                example: "-1KB | into filesize",
```

Comment:
```suggestion
                description: "Convert string with unit to filesize",
                example: "'-1KB' | into filesize",
```

Then, if you could change this to a string example, since we already have a filesize to filesize example right above this, thanks!

Reply:
Done

Reply:
Sorry, this was a two-line change. Could you add single quotes around `-1KB`, so that it is a string?